thrust/thrust/iterator/counting_iterator.h

```diff
@@ -247,6 +305,30 @@ inline _CCCL_HOST_DEVICE counting_iterator<Incrementable> make_counting_iterator
   return counting_iterator<Incrementable>(x);
 }
 
+// FIXME(bgruber): Sphinx fails to document the fancyiterators group if make_counting_iterator has overloads, so we
+// exclude them for now
+#ifndef _CCCL_DOXYGEN_INVOKED // Do not document
```

Comment:
I am very sad about this, but I found no other way to make the docs build pass :(

Reply:
😞